pants.toml

```diff
@@ -141,6 +141,11 @@ python-default = "3rdparty/python/user_reqs.lock"
 helm-post-renderer = "src/python/pants/backend/helm/subsystems/post_renderer.lock"
 helm-k8s-parser = "src/python/pants/backend/helm/subsystems/k8s_parser.lock"
 
+[python.resolves_to_interpreter_constraints]
+# Without setting this explicitly, the 'python-default' resolve uses the Pants default
```

Comment:
This seems like a crazy bug to exist this far along in time, but I feel like I've seen mention of it recentlyish in Slack? I cannot find an issue though.

Reply:
Ok, folks were silent; so I took this as either a known or unknown bug. I still could not find a relevant issue after a new search; so I filed: https://github.com/pantsbuild/pants/issues/17978